src/main/java/seedu/duke/logic/commands/circuit/TutorialCircuitCommand.java

```diff
@@ -54,10 +64,14 @@ public boolean continueTutorial(String command, Ui ui) throws DukeException {
         if (command.equals("exit")) {
             return false;
         }
-        
+        if (numOfCommandsDone >= 8 || !command.matches(orderOfInstructions[numOfCommandsDone] + "(.*)")) {
+            throw new DukeException("Please revise! Or you'll struggle with the module!\n" + "You got this! ᕦ(ò_óˇ)ᕤ\n"
+                    + "To exit Tutorial Mode, simply type 'exit' and press Enter.");
+        }
```

Comment:
Wait.. this will print once the person goes through 8 commands, and then print again because 8  > 6 commands done

Reply:
Adding on to this, how come there are two sets of `DukeException` being thrown?